catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/Validator.java

```diff
@@ -67,14 +68,9 @@ void validateTypes(List<QName> types, String version) throws CswException {
             return;
         }
 
-        if (types.size() == 1) {
-            QName typeName = types.get(0);
-            QName cswOutputSchema = new QName(CswConstants.CSW_OUTPUT_SCHEMA, CswConstants.CSW_RECORD_LOCAL_NAME);
-            QName gmdOutputSchema = new QName(GmdConstants.GMD_NAMESPACE, GmdConstants.GMD_LOCAL_NAME);
-            QName ebrimOutputSchema = new QName(CswConstants.EBRIM_SCHEMA, CswConstants.EBRIM_RECORD_LOCAL_NAME);
-            if (!typeName.equals(cswOutputSchema) && !typeName.equals(gmdOutputSchema) && !typeName.equals(ebrimOutputSchema)) {
-                throw createUnknownTypeException(types.get(0)
-                        .toString());
+        for (QName type : types) {
+            if (queryFilterTransformerHelper.getTransformer(type) == null) {
+                throw createUnknownTypeException(type.toString());
```

Comment:
_technically_ the service can be null/unregistered by the time the other code is called, but i don't think that will ever happen in real world scenario.